### ISSUE TITLE
Model Zoo - general use is matting or segment?

### ISSUE THREAD

**Issue by sandy-ssdut** [ZhengPeng7/BiRefNet#56]
Great work!

I noticed that the "general use task" model in model zoo use many matting dataset for training, such as TR-P3M-10k. The GT mask value is between [0,1].

So my question is the "general use task" model is trained for matting or segmentation? 
If this model is trained as matting task,  what is the difference in loss or other things from segmentation task like in DIS5K dataset?

Thanks very much!

**Comment by ZhengPeng7** [comment]
Thanks for your interest!
The datasets are mixed -- with both binary segmentation ones and soft ones.
However, concerning their number in the general datasets and the loss design for its training (mainly with BCE and IoU), it **leans more toward binary segmentation**.

**Comment by sandy-ssdut** [comment]
Thanks for your reply~

The matting mask GTs are not pre-preprocessed as binary one, is used between [0, 1] in training process, am I understanding this correctly?

If want to finetune the model to get better performance in matting task, like human hair. The only things that should do are:
1. use more matting dataset;
2. give lower weight for BCE and IoU, and higher weight for MAE, even add L1 loss
Is that correctly?

**Comment by ZhengPeng7** [comment]
Yeah. That's part of the reason I mean the model leans more toward binary segmentation. For matting, it might bring some conflict, but the majority of used datasets have 0 or 1 in GT, and 0.5 appears a few times; the weights of BCE and IoU are also different. Thus, practically, it works as usual. I also know the differences between losses of segmentation and matting, but currently no GPUs for me to conduct experiments on it.

**Comment by sandy-ssdut** [comment]
Got it. Thanks very much.
